src/qt/createwalletdialog.cpp

```diff
@@ -58,20 +58,18 @@ CreateWalletDialog::CreateWalletDialog(QWidget* parent) :
         ui->descriptor_checkbox->setChecked(checked);
         ui->encrypt_wallet_checkbox->setChecked(false);
         ui->disable_privkeys_checkbox->setChecked(checked);
-        // The blank check box is ambiguous. This flag is always true for a
-        // watch-only wallet, even though we immedidately fetch keys from the
-        // external signer.
-        ui->blank_wallet_checkbox->setChecked(checked);
+        ui->blank_wallet_checkbox->setChecked(false);
     });
 
     connect(ui->disable_privkeys_checkbox, &QCheckBox::toggled, [this](bool checked) {
         // Disable the encrypt_wallet_checkbox when isDisablePrivateKeysChecked is
         // set to true, enable it when isDisablePrivateKeysChecked is false.
         ui->encrypt_wallet_checkbox->setEnabled(!checked);
 
-        // Wallets without private keys start out blank
+        // Wallets without private keys cannot set blank
+        ui->blank_wallet_checkbox->setEnabled(!checked);
```

Comment:
If we disable `blank` checkbox when `disable_privkeys` is checked should we do the reverse?

Reply:
Done